src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,28 @@
 
 #include "t04_both2.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
 int t04_both2() {
+    set <int> s,s2,s3;
+    int n,x;
 
+    cin >> n;
+    for (int i = 0; i < n; i++) {
+        cin >> x;
+        s.insert(x);
+    };
+
+    cin >> n;
+    for (int i = 0; i < n; i++) {
+        cin >> x;
+        s2.insert(x);
+        if (s.find(x)) s3.insert(x);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Suspicious condition. The result of find() is an iterator, but it is not properly checked.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4763739)